keyserver/README.md

```diff
@@ -0,0 +1,35 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+```
+⇒  go install github.com/coniks-sys/coniks-go/keyserver/coniksserver
+⇒  coniksserver -h
+Usage of coniksserver:
+  -config string
+    	Path to the configuration file (default "config.toml")
```

Comment:
I'd like to have an option to specify the config file name, so that the developers don't have to use our default file name.